### PR TITLE
feat(new integration): stormly cloud mode destination (#2148)

### PR DESCRIPTION
**Pull request by ItsSudip**
## Description of the change

Onboarding new destination named stormly

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: src/v0/destinations/stormly/transform.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  if (!message.type) {
+    throw new InstrumentationError('Event type is required');
+  }
+
+  // set context.ip from request_ip if it is missing
+  if (!get(message, 'context.ip') && isDefinedAndNotNull(message.request_ip)) {
+    set(message, 'context.ip', message.request_ip);
+  }
+
+  const messageType = message.type.toLowerCase();
```
Comment (sanpj2292) [inline src/v0/destinations/stormly/transform.js:69]: Can we send `message.type` with a non-string value to transformer ?

Comment (ItsSudip) [reply src/v0/destinations/stormly/transform.js:69]: Yes, we can send message.type with a non-string value but the server is filtering that event.

Comment (sanpj2292) [reply src/v0/destinations/stormly/transform.js:69]: Ok

**Review by sanpj2292 — APPROVED** [review]

**Review by anantjain45823 — APPROVED** [review]
For Identify is their a reason we are not using `anonymousId` if `userId` is not given

**Comment by ItsSudip** [comment]
> For Identify is their a reason we are not using `anonymousId` if `userId` is not given

It is for tracking anonymous users

**Pull request merged**